### DISCUSSION TITLE
Using the extension to start and stop ec2 instance for remote development

### DISCUSSION THREAD

**Discussion by bwfinley** [aws/aws-toolkit-vscode#5083] [Q&A]
Hi, I'm working on a project which is using ec2 vm's as a back end. I would like to be able to start and stop my ec2 instance from the toolkit. I noticed that the features exist but are only accessible in development mode. I was wondering why this might be and if it could be changed?

Thanks

**Comment by justinmk3** [comment]
The EC2 feature hasn't been released yet. We are working to balance that with other priorities.
